[ChangeFeedProcessor]ExhaustAllRetriesBeforePollDelay (#35324)
Azure/azure-sdk-for-java: sdk/cosmos/azure-cosmos/CHANGELOG.md

#### Breaking Changes

#### Bugs Fixed

> * Fixed an issue for `ChangeFeedProcessor` that it should fetch all changes before delay the fetch based on configured `PollDelay`. - See [PR 35324](https://github.com/Azure/azure-sdk-for-java/pull/35324)

Reviewer: ```suggestion
* Fixed an issue with `ChangeFeedProcessor` to fetch all changes before delay based on configured `PollDelay`. - See [PR 35324](https://github.com/Azure/azure-sdk-for-java/pull/35324)
```

Author: updated
